jobs/elasticsearch_alerts/templates/cluster.alerts

```diff
@@ -0,0 +1,37 @@
+ALERT ElasticUp
+  IF elasticsearch_up != 1
+  FOR <%= p('elasticsearch_alerts.cluster_down.evaluation_time') %>
+  LABELS {
```

Comment:
To maintain consistency with all other alerts, can you please add a `service` label (`service = "elasticsearch"`)?